src/components/Notifications/notification.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { markNotificationAsRead } from "./../../services/notifications-service";
+
+/**
+ * A component to render one notification.
+ */
+const Notification = ({ notificationFromList }) => {
+
+    const [notification, setNotification] = useState(notificationFromList);
+    console.log(notification.id);
+
+    // create a notification message depending on the type of notification
+    var nstring;
+    if (notification.type == "FOLLOWS") {
```

Comment:
This if/else might grow quite a bit hypothetically if we add more types of notifications or turn into a long switch. How about an object literal/dictionary object with the type as a key and value as markup?